tests/trainers/test_detection.py

```diff
@@ -45,20 +45,27 @@ def forward(self, images: Any, targets: Any = None) -> Any:
         else:  # eval mode
             output = []
             for i in range(batch_size):
+                boxes = torch.zeros(10, 4, dtype=torch.float)
+                # Create xmax, ymax larger than 0.0
+                boxes[:, 2:4] = torch.FloatTensor(10, 2).uniform_(0.1, 0.9)
                 output.append(
                     {
-                        "boxes": torch.rand(10, 4),
+                        "boxes": boxes,
```

Comment:
This change alone is sufficient to increase coverage. Let's do the rest in a separate PR.